src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1557,20 +1577,6 @@ a.tooltip:hover::after {
 #src-sidebar div.files > a.selected {
 	background-color: var(--src-sidebar-background-selected);
 }
-#src-sidebar-toggle > button {
```

Comment:
I moved the block just under `#src-sidebar-toggle` rule as it made more sense like this.